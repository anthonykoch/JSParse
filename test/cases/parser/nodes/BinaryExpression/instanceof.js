10 instanceof 42
