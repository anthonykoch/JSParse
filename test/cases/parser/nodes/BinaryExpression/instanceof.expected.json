---
name: BinaryExpression - instanceof
description: parses simple instanceof binary
---

[
  {
    "type": "ExpressionStatement",
    "start": 0,
    "end": 16,
    "expression": {
      "type": "BinaryExpression",
      "operator": "instanceof",
      "start": 0,
      "end": 16,
      "left": {
        "type": "Literal",
        "value": "10",
        "raw": 10,
        "start": 0,
        "end": 2
      },
      "right": {
        "type": "Literal",
        "value": "42",
        "raw": 42,
        "start": 14,
        "end": 16
      }
    }
  }
]
